fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/ToolbarIntegration.kt

```diff
@@ -162,3 +162,61 @@ class DefaultToolbarIntegration(
         super.stop()
     }
 }
+
+/**
+ * [ToolbarIntegration] for the new redesigned toolbar.
+ * Removes tab counter button and menu button, otherwise same as [DefaultToolbarIntegration].
+ *
+ */
+@Suppress("LongParameterList")
+class RedesignToolbarIntegration(
```

Comment:
I can't make my mind on if adding a new integration is worth it. We have to duplicate 50 lines of code (constructor is 1 parameter different), and also probably have to create another one for the custom tab, right? If that's the experiment, the difference is logic would be not initializing tabs counter and hiding the menu. It looks like passing the experiment parameter into constructor and branching the logic would end up in less code. Might be even lesser clean up, because RedesignIntegration name doesn't make much sense if the experiment is successful and the design becomes the default, right? :)